src/jit/morph.cpp

```diff
@@ -4301,6 +4314,7 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
 #ifdef DEBUG
     if (verbose)
     {
+        JITDUMP("ArgTable for %d.%s after fgMorphArgs:\n", call->gtTreeID, GenTree::OpName(call->gtOper));
```

Comment:
This, and the similar change below, are because it's hard to figure out which argtable dump is for which calls when there are nested calls.